aws/v12.7.0/release.diff

```diff
@@ -0,0 +1,70 @@
+# Generated with:                                               <
+# devctl release create --name 12.6.0 --provider aws --base 12. <
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v12.6.0                                                 |    name: v12.7.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.2.3                                                     version: 1.2.3
+  - componentVersion: 1.0.2                                          - componentVersion: 1.0.2
+    name: cert-manager                                                 name: cert-manager
+    version: 2.3.0                                                     version: 2.3.0
```

Comment:
In https://github.com/giantswarm/giantswarm/issues/14284 we got asked to update to `2.3.2`. There is also [a "request" for aws](https://github.com/giantswarm/releases/blob/ac45b2ece0ffa6ebff630358a8eb2f0c2b04b626/aws/requests.yaml#L10-L17).

Reply:
done